nrf_security/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ if(CONFIG_BUILD_WITH_TFM)
     APPEND PROPERTY TFM_CMAKE_OPTIONS
       -DNRF_SECURITY_SETTINGS=\"$<TARGET_PROPERTY:nrf_security_target,CMAKE_ARGS>\"
       -DTFM_MBEDCRYPTO_CONFIG_PATH:STRING=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
-      -DMCUBOOT_MBEDCRYPTO_CONFIG_PATH=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
+      -DMCUBOOT_MBEDCRYPTO_CONFIG_FILEPATH=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>
```

Comment:
I recommend adding the same trick like we have done here: dbcb163, in case we add support for building MCUboot inside TF-M.

-DMCUBOOT_MBEDCRYPTO_CONFIG_FILEPATH:STRING=$<TARGET_PROPERTY:nrf_security_target,MBEDTLS_CFG_FILE>

Reply:
I saw that, but I don't understand why it is needed. Can you explain?

Reply:
explained ofline: it is to support relative filename instead of full path.